src/middleware/AuthMiddleware.ts

```diff
@@ -12,14 +12,25 @@ export class AuthMiddleware extends BaseMiddleware {
 
     public handler: RequestHandler = (req: Request, res: Response, next: NextFunction): void => {
 
+        req.session.ifNothing(() => console.log(`${req.url}: Session object is missing!`));
```

Comment:
what do we need the url for?

Reply:
To know where the app was trying to go

Reply:
ok, just incase this is not what we are expecting